### ISSUE TITLE
Cannot load .mpg videos to analyze

### ISSUE THREAD

**Issue by HallaMo** [DeepLabCut/DeepLabCut#2419]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I updated to the newest version of deeplabcut (2.3.7) but can no longer select .mpg videos to analyze. When trying to select videos in the GUI, my explorer only shows *.avi, *.mp4 and *.mov files. Is there a way I can fix this in order to avoid downgrading or converting all videos to one of the supported formats?

### Operating System

operating system: Windows 10 Pro

### DeepLabCut version

dlc version: 2.3.7 (GUI)

### DeepLabCut mode

single animal

### Device type

gpu nvidia

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
Hi @HallaMo we never had a mpg in the drop down menu. You can use the command line interface to easily add videos to the project as well, and you learn more! Check out: https://deeplabcut.github.io/DeepLabCut/docs/standardDeepLabCut_UserGuide.html#deeplabcut-in-the-terminal-command-line-interface

**Comment by HallaMo** [comment]
HI @MMathisLab,
yes, it was never in the drop down menu, but in the GUI, when adding videos, I was able to select them to analyse the position, whereas now they are not visible anymore. I added two snapshots for better understanding. In the new version, when the explorer opens to select videos, files are restricted to *.avi, *.mp4 and *.mov (I am unable to change it to show "all files").

![Screenshot (1)](https://github.com/DeepLabCut/DeepLabCut/assets/82214406/cae2959d-2a06-420f-a502-11f4e7a6940d)
![Screenshot (59)](https://github.com/DeepLabCut/DeepLabCut/assets/82214406/71d7e865-1d2b-4077-a470-e97031da6e2a)
